packages/story-editor/src/utils/cleanForSlug.ts

```diff
@@ -53,6 +56,6 @@ export default function cleanForSlug(string, isEditing = false) {
       .map((s) => (isEditing ? s : s.replace(/[^\p{L}\p{N}_-]+/gu, '')))
       .map((s) => (isEditing ? s : s.replace(/--+/g, '-')))
       .map((s) => s.toLowerCase())
-      .pop()
+      .pop() as string
```

Comment:
Hmm, I expected that `.pop()` on a `[string]` tuple would be typed as always returning a string, but it's still typed as `string | undefined`. Annoying :/

Reply:
I do that here - https://github.com/GoogleForCreators/web-stories-wp/pull/12737/files#diff-2c4c08b4833fb80bb4751a4252439074bd9c084802deafa659abd3a7d7cbcddfR45